lookyloo/modules/misp.py

```diff
@@ -271,6 +271,8 @@ def lookup(self, node: URLNode, hostnode: HostNode) -> dict[str, set[str]] | dic
                     # NOTE: We have MISPAttribute in that list
                     for a in attributes:
                         to_return[a.event_id].add(a.value)  # type: ignore[union-attr,index]
+                        if time:
+                            to_return[a.event_id].add(a.timestamp)
```

Comment:
When `time` is `True`, we want to return a `tuple[str, str]` instead of `set[str]` in the dictionary, it will make it a lot easier to use if we have more than one attribute matching in the same event (not relevant for the takedown system, but relevant on other cases).